packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -221,19 +216,26 @@ if (!process.env.PRIVATE_KEY_USER) {
 const localWallet = new Wallet(process.env.PRIVATE_KEY_CUSTOM)
 const userWallet = new Wallet(process.env.PRIVATE_KEY_USER)
 
-async function deployERC20ToL1() {
-  console.log('Deploying ERC20 to L1...')
-  const contract = new TestERC20__factory().connect(
-    localWallet.connect(parentProvider)
+async function deployERC20ToParentChain() {
+  console.log('Deploying ERC20...')
+  const signer = localWallet.connect(parentProvider)
+  const factory = new ContractFactory(contractAbi, contractByteCode, signer)
+  const l1TokenContract = await factory.deploy(
+    'Test Arb Token',
+    'TESTARB',
+    parseEther('100').toString()
   )
-  const token = await contract.deploy()
-  await token.deployed()
-
-  return token
+  console.log('Deployed ERC20:', {
+    symbol: await l1TokenContract.symbol(),
+    address: l1TokenContract.address,
+    supply: formatEther(
```

Comment:
why is it `formatEther` not `formatUnits` when this is an ERC20?

Reply:
Updated to `formatUnits` and `parseUnits` (with Ether utils we didn't need to specify decimals since it's 18 by default, but now we do)